core/src/main/java/org/apache/iceberg/SnapshotSummary.java

```diff
@@ -47,6 +47,7 @@
   public static final String PUBLISHED_WAP_ID_PROP = "published-wap-id";
   public static final String SOURCE_SNAPSHOT_ID_PROP = "source-snapshot-id";
   public static final String REPLACE_PARTITIONS_PROP = "replace-partitions";
+  public static final String EXTRA_METADATA_PREFIX = "extra-metadata.";
```

Comment:
happy to change if a different prefix string would be more suitable

Reply:
Sounds good to me. What about `context.` or `snapshot.property.`? I want it to be somewhat obvious what the prefix indicates, but as short as possible.

Reply:
`snapshot.property` sounds good. I'll make changes.